.github/workflows/ios-build.yml.bk

```
@@ -0,0 +1,54 @@
+name: iOS Build
+
+on:
+  push:
+    branches: [] # TODO: Setting is still
```

actionlint (1.7.10)

[error] "branches" section should not be empty

(syntax-check)